lib/bulma_form_builder/inputs/submit.rb

```diff
@@ -0,0 +1,23 @@
+module BulmaFormBuilder
+  module Inputs
+    module Submit
+      extend ActiveSupport::Concern
+
+      include Base
+
+      included do
+        def submit_with_bulma(value = nil, options = {})
+          if value.is_a?(Hash)
+            options = value
+            value = nil
+          end
+          options[:class] = options[:class].present? ? Array(options[:class]) : []
+          options[:class] += [:button, :'is-primary']
```

Comment:
It should not always be an `is-primary` class.

Reply:
OK. This can be removed when the previous line is changed.